src/index.js

```diff
@@ -1,11 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
+/* eslint-disable */
```

Comment:
- [ ] Kindly remove this `eslint-disable` in all of your component files to make sure that linters are working and you are following best practices in all of your components. Please try not to disable linter checks under any circumstances unless you get specific go-ahead from Microverse.